htdocs/modulebuilder/template/test/phpunit/MyObjectTest.php

```diff
@@ -81,7 +82,7 @@ public function __construct($name = '')
 	 *
 	 * @return void
 	 */
-	public static function setUpBeforeClass()
+	public static function setUpBeforeClass(): void
```

Comment:
As the transition from PHP7.0 to PHP7.1+ is not yet complete by a lot of users, for the moment, we must avoid the use of the 
": type"
We will continue to wait a little bit that the nb of users with php 7.0 is lower (A lot of users first upgrade their Dolibarr to be able to upgrade their PHP, so having a Dolibarr not hanging on fatal error with PHP 7.0 is still important) .